### PR TITLE
Added new guides, updated descriptions

### PR DESCRIPTION
**Pull request by OhanNanen-MS**
I added a new guide for AD FS migration. Other guides had description and title updates due to recent changes. Updated links and meta description.

### CODE REVIEW

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 1]
```diff
@@ -83,13 +83,14 @@ Advanced deployment guides in the admin center require authentication to a Micro
...
-||[Secure your cloud apps with Single Sign on (SSO) guide](https://go.microsoft.com/fwlink/?linkid=2224689)|The **Secure your cloud apps with Single Sign on (SSO) guide** helps IT admins configure third-party cloud apps with single sign-on, which reduces or eliminates sign-in prompts.|
+|[Azure AD setup guide](https://go.microsoft.com/fwlink/?linkid=2223229)|[Azure AD setup guide](https://go.microsoft.com/fwlink/?linkid=2224193)|The **Azure AD setup guide** provides information to ensure your organization has a strong security foundation. In this guide you'll set up initial features, like Azure Role-based access control (Azure RBAC) for admins, Azure AD Connect for your on-premises directory, and Azure AD Connect Health, so you can monitor your hybrid identity's health during automated syncs.<br>It also includes essential information on enabling self-service password resets, conditional access and integrated third party sign-on including optional advanced identity protection and user provisioning automation.|
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
|[Azure AD setup guide](https://go.microsoft.com/fwlink/?linkid=2223229)|[Azure AD setup guide](https://go.microsoft.com/fwlink/?linkid=2224193)|The **Azure AD setup guide** provides information to ensure your organization has a strong security foundation. In this guide you'll set up initial features, like Azure Role-based access control (Azure RBAC) for admins, Azure AD Connect for your on-premises directory, and Azure AD Connect Health, so you can monitor your hybrid identity's health during automated syncs.<br>It also includes essential information on enabling self-service password resets, conditional access, and integrated third party sign-on including optional advanced identity protection and user provisioning automation.|
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 2]
```diff
@@ -83,13 +83,14 @@ Advanced deployment guides in the admin center require authentication to a Micro
...
 |[Plan your self-service password reset (SSPR) deployment guide](https://go.microsoft.com/fwlink/?linkid=2223231)|[Plan your self-service password reset (SSPR) deployment guide](https://go.microsoft.com/fwlink/?linkid=2224781)|Give users the ability to change or reset their password independently, if their account is locked, or they forget their password without the need to contact a helpdesk engineer.<br>Use the **Plan your self-service password reset (SSPR) deployment guide** to receive relevant articles and instructions for configuring the appropriate Azure portal options to help you deploy SSPR in your environment.|
+|[Migrate from AD FS to Microsoft Azure AD](https://go.microsoft.com/fwlink/?linkid=2229256)|[Migrate from AD FS to Microsoft Azure AD](https://go.microsoft.com/fwlink/?linkid=2225005)|In **Migrate from ADFS to Microsoft Azure AD** we offer custom guidance for migrating from Active Directory Federation Services (AD FS) to Azure AD. You'll first answer a few questions about your AD FS infrastructure. Then implement either pass-through authentication (PTA) or password hash sync (PHS) to give users a streamlined experience while accessing your organization's apps.|
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
|[Migrate from AD FS to Microsoft Azure AD](https://go.microsoft.com/fwlink/?linkid=2229256)|[Migrate from AD FS to Microsoft Azure AD](https://go.microsoft.com/fwlink/?linkid=2225005)|In **Migrate from AD FS to Microsoft Azure AD** we offer custom guidance for migrating from Active Directory Federation Services (AD FS) to Azure AD. You'll first answer a few questions about your AD FS infrastructure. Then implement either pass-through authentication (PTA) or password hash sync (PHS) to give users a streamlined experience while accessing your organization's apps.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @OhanNanen-MS, kindly commit the minor suggested changes. Many thanks.

**Comment by OhanNanen-MS** [comment]
@JohanFreelancer9 Changes committed. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**